components/org.wso2.carbon.event.outbound.adapter.websubhub/src/main/java/org/wso2/carbon/event/outbound/adapter/websubhub/WebSubHubAdapterService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.event.outbound.adapter.websubhub;
+
+import org.wso2.carbon.event.outbound.adapter.websubhub.exception.WebSubAdapterException;
+import org.wso2.carbon.event.outbound.adapter.websubhub.model.EventPayload;
+
+/**
+ * The WebSubHub event adapter service is used to publish notification events into the intermediate hub.
+ */
+public interface WebSubHubAdapterService {
+
+    void publish(EventPayload payload, String topicSuffix, String eventUri) throws WebSubAdapterException;
```

Comment:
add method descriptions

Reply:
addressed in https://github.com/wso2-extensions/identity-outbound-adapters/pull/1/commits/3cb452bb085bfec3e826b7c47723c3d161577f0a